dozer-types/src/tests/dozer_yaml_deserialize.rs

```diff
@@ -156,13 +156,13 @@ fn error_missing_field_inner() {
       columns:
       - id
       - email
-      - phone   
+      - phone
   "#;
     let deserialize_result = serde_yaml::from_str::<Config>(input_config);
     let error = deserialize_result.err();
     assert!(error.is_some());
     assert!(error
         .unwrap()
         .to_string()
-        .starts_with("connections[0].config: missing field `password`"));
+        .starts_with("sources[0]: missing connection ref"));
```

Comment:
I think this test is meant to check that, when `password` is missing, it's properly reported.

Reply:
The reason why we are getting that is due to, cause we change that to a optional variable, it will create the Config variable. It used to fail without creating them.

Reply:
Got it. Then maybe change the test name? It's actually testing `missing connection ref` now.

Reply:
got it!